### PR TITLE
buttonBorderUpdate, added constraint to borderView bottom to buttonSV…

### PR DESCRIPTION
**Pull request by bhtran**
- Forgot to add constraint to borderView to buttonSV
- Height is now dependent on pixel width

### CODE REVIEW

**Comment by bhtran** [comment]
@codecaffeine ready to be looked at

File: URBNSwiftAlert/Classes/AlertView.swift [hunk 1]
```diff
@@ -126,6 +128,7 @@ extension AlertView {
         buttonsSV.distribution = .fillEqually
         buttonsSV.spacing = configuration.styler.button.spacing
         stackView.addArrangedSubviews(separatorBorderView, buttonsSV.wrapInNewView(with: configuration.styler.button.containerInsetConstraints))
+        separatorBorderView.bottomAnchor.constraint(equalTo: buttonsSV.topAnchor).isActive = true
```
Comment (codecaffeine) [inline URBNSwiftAlert/Classes/AlertView.swift]: I wouldn’t just constrain this within the stack view. I would create a new stack view with 0 spacing and add these two views in them

**Review by codecaffeine — CHANGES_REQUESTED** [review]

File: URBNSwiftAlert/Classes/AlertView.swift [hunk 2]
```diff
@@ -125,7 +127,14 @@ extension AlertView {
         buttonsSV.axis = configuration.actions.count < 3 ? configuration.styler.button.layoutAxis : .vertical
         buttonsSV.distribution = .fillEqually
         buttonsSV.spacing = configuration.styler.button.spacing
-        stackView.addArrangedSubviews(separatorBorderView, buttonsSV.wrapInNewView(with: configuration.styler.button.containerInsetConstraints))
+
+        let borderButtonSV = UIStackView(arrangedSubviews: [separatorBorderView, buttonsSV.wrapInNewView(with: configuration.styler.button.containerInsetConstraints)])
+        borderButtonSV.spacing = 0
```
Comment (codecaffeine) [inline URBNSwiftAlert/Classes/AlertView.swift]: default is zero

File: URBNSwiftAlert/Classes/AlertView.swift [hunk 3]
```diff
@@ -125,7 +127,14 @@ extension AlertView {
...
+        borderButtonSV.axis = .vertical
+
+        separatorBorderView.bottomAnchor.constraint(equalTo: buttonsSV.topAnchor).isActive = true
```
Comment (codecaffeine) [inline URBNSwiftAlert/Classes/AlertView.swift]: This isn’t needed

**Review by codecaffeine — CHANGES_REQUESTED** [review]

File: URBNSwiftAlert/Classes/AlertView.swift (hunk 3 above)
Comment (bhtran) [reply URBNSwiftAlert/Classes/AlertView.swift]: Addressed

File: URBNSwiftAlert/Classes/AlertView.swift (hunk 2 above)
Comment (bhtran) [reply URBNSwiftAlert/Classes/AlertView.swift]: Addressed

**Review by codecaffeine — APPROVED** [review]

**Pull request merged**